packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -310,8 +312,25 @@ function requestSignature(
   controller: AbortController,
   logger: Logger
 ): Promise<FetchResponse> {
-  logger.debug({ signer: service.url }, `Requesting partial sig`)
-  const url = service.url + PARTIAL_SIGN_MESSAGE_ENDPOINT
+  const signerCircuitBreaker = new CircuitBreaker(parametrizedSignatureRequest)
+  signerCircuitBreaker.fallback(() =>
+    parametrizedSignatureRequest(service.fallbackUrl, request, controller, logger)
```

Comment:
We should allow `fallbackUrl` to not be specified for some signers.

Reply:
Added this as an optional parameter.